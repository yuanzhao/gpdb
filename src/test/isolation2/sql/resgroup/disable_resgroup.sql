-- reset the GUC and restart cluster.
-- start_ignore
! gpconfig -r gp_resource_manager;
! gpstop -rai;
-- end_ignore

SHOW gp_resource_manager;
