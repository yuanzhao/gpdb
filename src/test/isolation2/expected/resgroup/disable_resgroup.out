-- reset the GUC and restart cluster.
-- start_ignore
! gpconfig -r gp_resource_manager;

! gpstop -rai;
-- end_ignore

SHOW gp_resource_manager;
gp_resource_manager
-------------------
queue              
(1 row)
